schema-loader/sample/import_schema_sample.json

```diff
@@ -1,6 +1,10 @@
 {
   "sample_namespace1.sample_table1": {
-    "transaction": true
+    "transaction": true,
+    "override-columns-type": {
```

Comment:
By setting a columns and the desired data type inside the `override-columns-type` JSON object of the import table schema file , user can override the default type mapping.
This is only useful for some type-related types on Mysql and Oracle now.